src/Commands/SystemCommands/GenericmessageCommand.php

```diff
@@ -23,13 +24,36 @@ class GenericmessageCommand extends SystemCommand
     protected $name = 'Genericmessage';
     protected $description = 'Handle generic message';
     protected $version = '1.0.1';
+    protected $need_mysql = true;
     /**#@-*/
 
     /**
-     * {@inheritdoc}
+     * Execution if MySQL is required but not available
+     *
+     * @return boolean
      */
-    /*public function execute()
+    public function executeNoDB()
     {
+        //Do nothing
+        return true;
+    }
 
-    }*/
+    /**
+     * Execute command
+     *
+     * @return boolean
+     */
+    public function execute()
+    {
+        //System command, fetch command to execute if track exist
+        $message = $this->getMessage();
+        $chat_id = $message->getChat()->getId();
+        $user_id = $message->getFrom()->getId();
+        //Fetch Track if exist
+        $command = (new Conversation($user_id, $chat_id))->getConversationCommand();
+        if (! is_null($command)) {
+            return $this->telegram->executeCommand($command, $this->update);
+        }
+        return true;
```

Comment:
Must return `ServerResponse`.

For an empty server response, just use:
```php
return new ServerResponse(['ok' => true, 'result' => true], null);
```

We should add a static method to the `ServerResponse` class to help with this, for example:
```php
public static function emptyResponse() {
    return new ServerResponse(['ok' => true, 'result' => true], null);
}
```
Then we could just call
```php
return ServerResponse::emptyResponse();
```